File: python-venv/only_duplicates/kata_solution/only_duplicates.py
def only_duplicates(string):
    counts = {}
    for i in range(0, len(string), 1):
        if string[i] in counts:
            counts[string[i]] += 1
        else:
            counts[string[i]] = 1

    output = ""
    for j in range(0, len(string), 1):
        if counts[string[j]] > 1:
            output += string[j]

    return output


#https://www.codewars.com/kata/5a1dc4baffe75f270200006b/train/python

File: only-duplicates/onlyDuplicates.js
const onlyDuplicates = (string) => {
    const counts = {};

    string.split('').forEach((element) => {
        if(counts[element] !== undefined){
            counts[element]++;
        } else {
            counts[element] = 1;
        };
    });

    let output = '';
    for(let i = 0; i < string.length; i++){
        if(counts[string[i]] > 1){
            output += string[i];
        };
    };

    return output;
};

module.exports = { onlyDuplicates };

//https://www.codewars.com/kata/5a1dc4baffe75f270200006b/train/javascript